lib/ansible/modules/cloud/misc/virt.py

```diff
@@ -23,14 +23,15 @@
 options:
   name:
     description:
-      - name of the guest VM being managed. Note that VM must be previously
-        defined with xml.
-    required: true
+      - name of the guest VM being managed. Almost always required (see examples). When defining a new VM,
+        name is taken from xml.
```

Comment:
In previous version, name is required parameter. So add a note here stating- 

```
C(name) is optional parameter from version 2.7 and onwards.
```

Reply:
Sure!
It actually was not really required. You could run things like command: list_vms without name.